integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/merge/MergeTestCase.scala

```diff
@@ -869,10 +910,15 @@ class MergeTestCase extends QueryTest with BeforeAndAfterAll {
     CarbonProperties.getInstance().addProperty("carbon.enable.auto.load.merge", "false")
   }
 
-  private def getDeleteDeltaFileCount(tableName: String, segment: String): Int = {
+  private def getDeleteDeltaFileCount(tableName: String,
+      segment: String,
```

Comment:
no need to change the method signature, you are already having the carbonTable object, check from it whether its partition table or not.